idna/codec.py

```diff
@@ -1,3 +1,4 @@
+# type: ignore
```

Comment:
Will remove this once I fix `idna.codec`, the types in this file don't line up and cause the codecs to not work.